src/integrationTest/java/com/hivemq/helmcharts/util/OperatorHelmChartContainer.java

```diff
@@ -0,0 +1,77 @@
+package com.hivemq.helmcharts.util;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.testcontainers.containers.BindMode;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.images.builder.ImageFromDockerfile;
+import org.testcontainers.k3s.K3sContainer;
+import org.testcontainers.utility.DockerImageName;
+
+import java.io.File;
+import java.io.IOException;
+
+public class OperatorHelmChartContainer extends K3sContainer {
+    public final static int mqttPort = 1883;
+
+    private OperatorHelmChartContainer(final @NotNull String k3sVersion,
+                                       final @NotNull File dockerfile,
+                                       final @NotNull File mountPath,
+                                       final @NotNull String containerPath) throws IOException {
+        super(getAdHocImageName(k3sVersion, dockerfile));
+        super.addFileSystemBind(mountPath.getCanonicalPath(),containerPath, BindMode.READ_WRITE);
```

Comment:
Please use `withCopyFileToContainer()` instead of `addFileSytemBind()` if possible

Reply:
We need several files from the path, not only one or two files, withClasspathResourceMapping is also possible, not sure about the advantage of one over the other

Reply:
`withCopyFileToContainer()` is more stable on different host systems compared to mappings/mounts